CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing
```

Comment:
@anthonyharrison, this is awesome. Thank you for making such a comprehensive submission and adding so much. Reviewing now.

Reply:
Thank you, @anthonyharrison. This is great work.